pkg/cmd/server/start/master_args.go

```diff
@@ -344,6 +344,17 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 	configapi.SetProtobufClientDefaults(config.MasterClients.OpenShiftLoopbackClientConnectionOverrides)
 	configapi.SetProtobufClientDefaults(config.MasterClients.ExternalKubernetesClientConnectionOverrides)
 
+	// Allow any service account in openshift-infra to participate in bootstrapping by default.
+	// This is experimental.
+	if km := config.KubernetesMasterConfig; km != nil {
+		if len(km.ControllerArguments) == 0 {
+			km.ControllerArguments = configapi.ExtendedArguments{}
+			km.ControllerArguments["cluster-signing-cert-file"] = []string{admin.DefaultCertFilename(args.ConfigDir.Value(), admin.CAFilePrefix)}
+			km.ControllerArguments["cluster-signing-key-file"] = []string{admin.DefaultKeyFilename(args.ConfigDir.Value(), admin.CAFilePrefix)}
+			km.ControllerArguments["insecure-experimental-approve-all-kubelet-csrs-for-group"] = []string{"system:serviceaccounts:openshift-infra"}
```

Comment:
Why isn't this permissions based anyway.  Why hardcoded groups?  This doesn't look right.